pkg/thanos/statefulset.go

```diff
@@ -37,6 +37,7 @@ const (
 	defaultPortName           = "web"
 	defaultRetention          = "24h"
 	defaultEvaluationInterval = "15s"
+	defaultReplicaLabelName   = "ruler_replica"
```

Comment:
I think we should default this to "thanos_ruler_replica" just to make it clearer, WDYT?

Reply:
It is definitely better. Thanks!